.changeset/eleven-poems-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@smithy/util-waiter": patch
+---
+
+this validation is fixed for all waiters
```

Comment:
This message appears in the changelog, so "this validation" is going to be ambiguous. 

The message can be "fix range validation in waiters".